curriculum/challenges/english/25-front-end-development/quiz-backend-javascript/66f1b11026394bcdb7c42473.md

```diff
@@ -17,375 +17,374 @@ Answer all of the questions below correctly to pass the quiz.
 
 #### --text--
 
-Placeholder question
+Which command initializes a new package.json file in npm?
 
 #### --distractors--
 
-Placeholder distractor 1
+npm create
 
 ---
 
-Placeholder distractor 2
+npm add
 
 ---
 
-Placeholder distractor 3
+npm start
 
 #### --answer--
 
-Placeholder answer
+npm init
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What is the primary purpose of `npm`?
 
 #### --distractors--
 
-Placeholder distractor 1
+To run a web server
 
 ---
 
-Placeholder distractor 2
+To bundle code for production
 
 ---
 
-Placeholder distractor 3
+To compile JavaScript
 
 #### --answer--
 
-Placeholder answer
+To manage packages for Node.js
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which JavaScript engine is used by Node.js?
 
 #### --distractors--
 
-Placeholder distractor 1
+JavaScriptCore
 
 ---
 
-Placeholder distractor 2
+SpiderMonkey
 
 ---
 
-Placeholder distractor 3
+Chromium
 
 #### --answer--
 
-Placeholder answer
+V8
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+In Express.js, what middleware function is used to parse incoming JSON payloads?
 
 #### --distractors--
 
-Placeholder distractor 1
+app.useJSON()
 
 ---
 
-Placeholder distractor 2
+app.parseJSON()
 
 ---
 
-Placeholder distractor 3
+express.bodyParser()
 
 #### --answer--
 
-Placeholder answer
+express.json()
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What is Express.js primarily used for?
 
 #### --distractors--
 
-Placeholder distractor 1
+Creating mobile applications
 
 ---
 
-Placeholder distractor 2
+Creating desktop applications
 
 ---
 
-Placeholder distractor 3
+Database management
 
 #### --answer--
 
-Placeholder answer
+Building web applications and APIs
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What is a major feature of Deno compared to Node.js?
 
 #### --distractors--
 
-Placeholder distractor 1
+JavaScript syntax only
 
 ---
 
-Placeholder distractor 2
+Support for CommonJS modules
 
 ---
 
-Placeholder distractor 3
+Requires a browser to run
 
 #### --answer--
 
-Placeholder answer
+Built-in TypeScript support
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What is Bun primarily designed for?
```

Comment:
Same as with Deno.

Reply:
Deno and Bun are within the topics listed in the issue, so I guess they are fine?

Reply:
I guess so. Just seems a little odd to me if the quiz is about Node and Express.